src/gflownet/train.py

```diff
@@ -238,6 +238,8 @@ def run(self, logger=None):
         self.model.to(self.device)
         self.sampling_model.to(self.device)
         epoch_length = max(len(self.training_data), 1)
+        valid_freq = self.hps.get("validate_every", 0)
+        ckpt_freq = self.hps.get("checkpoint_every", valid_freq)
```

Comment:
What does passing in the `valid_freq` here do?

Reply:
It makes the checkpoint frequency default to the value of `validate_every` (`hps` is a `dict`, `a_dict.get(key, default)` returns `default` when `key not in a_dict`). I'll add a comment.